test/rubocop/cop/mixin/minitest_exploration_helpers_test.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class MinitestExplorationHelpersTest < Minitest::Test
+  module Helper
+    extend RuboCop::Cop::MinitestExplorationHelpers
+    class << self
+      public :test_case?
+    end
+  end
+
+  SOURCE = <<~RUBY
+    class FooTest < Minitest::Test
+      def not_test_case; end
+      def test_with_arguments(arg); end
+      def test_public; end
+
+      protected
+      def test_protected; end
+
+      private
+      def test_private; end
+    end
+  RUBY
+
+  def test_test_case_p_returns_true_for_test_case
```

Comment:
What does the `p` mean here?

Reply:
"predicate". It's a replacement for `?` at the end of the method name. I saw this convention in the ruby source code. For example, https://github.com/ruby/ruby/blob/a9bcc058bb4cd954e8406c93debb876983627ca9/array.c#L8851.

Reply:
Even if `p` is used in the product code, the `test_` descriptions might be easier to understand with `predicate`.
On a side note, `p` reminds me of LISP :-)